rusk-wallet/src/bin/command/history.rs

```diff
@@ -270,9 +283,31 @@ pub(crate) async fn moonlight_history(
                     amount += event_amount;
                 }
                 BlockData::PhoenixTransactionEvent(_) => {
+                    // This comes up in a phoenix to moonlight conversion.
+                    // In this conversion, a `PhoenixTransactionEvent`
+                    // and `ConvertEvent` are emitted.
+                    //
                     // This case has already been handled in
                     // `transaction_from_notes`
                 }
+                BlockData::DepositEvent(_) => {
+                    // This comes up for stake events: the value
+                    // staked is deposited in the stake contract.
+                    // Nothing needs to be done in this case because the
+                    // other two events, moonlight transaction event and
+                    // stake event handle everything.
```

Comment:
A deposit-event is triggered when funds are deposited in a contract. Funds can be deposited in a contract either through phoenix or through moonlight. The deposit that happens when staking is one example for this mechanism, but deposits are not limited to the stake-contract.